src/GraphQL/FieldConfigGenerator/Helper/ImageGallery.php

```diff
@@ -85,7 +85,7 @@ public function resolve($value = null, $args = [], $context = [], ResolveInfo $r
                     }
 
                     if ($image instanceof Asset) {
-                        $data = new \ArrayObject();
+                        $data = new \ArrayObject(['data' => null]);
```

Comment:
Please switch to ElementDescriptor.
see https://github.com/pimcore/data-hub/issues/15